config/tracing.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"errors"
+)
+
+type TracingConfig struct {
+	Enabled                 bool                 `yaml:"enabled" default:"false"`
+	Attributes              Map                  `yaml:"attributes"`
+	Opentelemetry           *OpenTelemetryConfig `yaml:"opentelemetry"`
+	ServiceName             string               `yaml:"service_name" default:"WebhookX" envconfig:"SERVICE_NAME"`
+	CapturedRequestHeaders  []string             `yaml:"captured_request_headers" envconfig:"CAPTURED_REQUEST_HEADERS"`
+	CapturedResponseHeaders []string             `yaml:"captured_response_headers" envconfig:"CAPTURED_RESPONSE_HEADERS"`
+	SafeQueryParams         []string             `yaml:"safe_query_params" envconfig:"SAFE_QUERY_PARAMS"`
+	SamplingRate            float64              `yaml:"sampling_rate" default:"1.0" envconfig:"SAMPLING_RATE"`
+}
+
+type OpenTelemetryConfig struct {
+	Protocol OtlpProtocol `yaml:"protocol" envconfig:"PROTOCOL" default:"http/protobuf"`
+	Endpoint string       `yaml:"endpoint" envconfig:"ENDPOINT" default:"http://localhost:4318/v1/metrics"`
+	Headers  Map          `yaml:"headers" envconfig:"HEADERS"`
```

Comment:
Let's remove Headers as it can be configured via  
 [OTEL_EXPORTER_OTLP_TRACES_HEADERS](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_traces_headers)